### DISCUSSION TITLE
How to Clear/Reset your ZeroTier Address in home assistant, have cloned a device and you want to prevent it from using the same address?

### DISCUSSION THREAD

**Discussion by Mivlz** [hassio-addons/app-zerotier#127] [Q&A] [answered]
How to Clear/Reset your ZeroTier Address in home assistant, have cloned a device and you want to prevent it from using the same address?

**Comment by frenck** [comment, marked as answer]
You can delete the `/ssl/zerotier` folder and restart the add-on to achieve that.

../Frenck

**Comment by Mivlz** [comment]
Спасибо огромное
